lua/nvim-treesitter/install.lua

```diff
@@ -42,6 +42,26 @@ local function get_job_status()
     .. "]"
 end
 
+---@param lang string
+---@return function
+local function reattach_if_possible_fn(lang)
+  return function()
+    for _, buf in ipairs(vim.api.nvim_list_bufs()) do
+      if parsers.get_buf_lang(buf) == lang then
+        vim._ts_remove_language(lang)
+        local ok = pcall(vim.treesitter.language.require_language, lang)
```

Comment:
TODO: this could silence an error during installation. During uninstallation, failure is expected unless there's another parser.

Reply:
arguably, we should only require the `lang` if it was required before the `remove`